### PR TITLE
Feature mpu class init

### PR DESCRIPTION
**Pull request by renskekuip1**

### CODE REVIEW

File: src/MPU9250_Interface.cpp [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+ * @brief     MPU9250Interface Class
+ * @author    Renske Kuip
+ * @license   See LICENSE
+ */
+
+#include "mpu9250_interface.hpp"
+#include "memory_map.hpp"
+
+void MPU9250Interface::init() {
+    hwlib::cout << hwlib::right << "Initializing MPU9250..." << hwlib::endl;
```
Comment (elsavira) [inline src/MPU9250_Interface.cpp]: Technically, debug prints are not allowed in production code.

File: src/MPU9250_Interface.cpp [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    /*for (;;) {
```
Comment (elsavira) [inline src/MPU9250_Interface.cpp]: Remove these comments, or mention they're temporary.

File: src/MPU9250_Interface.cpp [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    i2c.read(MPUAddr, data, 12);
+
+    gyroValue[0] = (((int16_t)data[6] << 8) | (int16_t)data[7]);
+    gyroValue[1] = (((int16_t)data[8] << 8) | (int16_t)data[9]);
+    gyroValue[2] = (((int16_t)data[10] << 8) | (int16_t)data[11]);
+
+    printValuesX_Y_Z(gyroValue);
+}
+
+void MPU9250Interface::getMagn() {
```
Comment (elsavira) [inline src/MPU9250_Interface.cpp]: Use full names for this function. getMagnetometer() is fine.

File: src/MPU9250_Interface.hpp [hunk 4]
```diff
@@ -8,10 +8,70 @@
...
 
-class MPU9250Interface
-{
-public:
-    MPU9250Interface(){}
+#include "wrap-hwlib.hpp"
+
+class MPU9250Interface {
+  public:
+    MPU9250Interface(hwlib::pin_oc &scl, hwlib::pin_oc &sda, const int8_t MPUAddr)
```
Comment (elsavira) [inline src/MPU9250_Interface.hpp]: add Doxygen for the constructor.

File: src/Position_Module.hpp [hunk 5]
```diff
@@ -8,16 +8,16 @@
 #ifndef POSITION_MODULE_HPP
 #define POSITION_MODULE_HPP
 
-#include "MPU9250_Interface.hpp"
+//#include "MPU9250_Interface.hpp"
```
Comment (elsavira) [inline src/Position_Module.hpp]: Temporary comment? Mention if so.

**Review by elsavira — CHANGES_REQUESTED** [review]
Some changes are needed. Make sure to merge the current dev into your branch before requesting review.

**Comment by renskekuip1** [comment]
image

**Comment by elsavira** [comment]
Travis build test fails on HWLIB/ARDUINO items. 

Lizard and Clang-format tests succesfully completed. There are no unit-tests required for this feature.

Code builds properly, so accepting this request.

**Pull request merged**